simcore/simulation/executable/CMakeLists.txt

```diff
@@ -16,10 +16,19 @@ if (OMP)
 endif()
 
 if (GRAPH)
-  find_package(glfw3 REQUIRED)
-  find_package(glew REQUIRED)
-  find_package(OpenGL REQUIRED)
-  set(LIB ${LIB} GLEW::GLEW ${GLFW3_LIBRARIES} ${OPENGL_gl_LIBRARY} glfw )
+  find_package(glfw3 REQUIRED)# HINTS /usr/include/GLFW)
+  if (${WINDOWS})
+    set(GLEW_LIB -lGL -lGLU -lGLEW -ldl -lX11 -lpthread -lXxf86vm -lXi -lm -lXcursor -lXinerama)
+    set(GLEW_INCLUDE_DIRS /usr/include/)
+  else()
+    find_package(glew REQUIRED)
+  endif() 
+  find_package(OpenGL REQUIRED)  
+  if (${WINDOWS}) 
+    set(LIB ${LIB} ${GLEW_LIB} ${GLFW3_LIBRARIES} ${OPENGL_gl_LIBRARY} glfw)
+  else() 
+    set(LIB ${LIB} GLEW::GLEW ${GLFW3_LIBRARIES} ${OPENGL_gl_LIBRARY} glfw)
+  endif()
```

Comment:
These two if/else statement blocks can be cleaned up into one if/else statement for more readability